provisioning/miniprov/hare_mp/utils.py

```diff
@@ -128,3 +144,19 @@ def execute(cmd: List[str], env=None) -> str:
             f'Command output: {err}')
 
     return out
+
+
+def execute_no_communicate(cmd: List[str], env=None,
+                           working_dir: str = '/var/log/cortx/hare',
+                           out_file=subprocess.PIPE):
+    process = subprocess.Popen(cmd,
```

Comment:
I don't quite get how is it supposed to work.

See, the end user will invoke `hare_setup start` to start hax. But in fact the first script spawns a new process that runs hax. Is the `hare_setup` process finishes keeping `hax` running in the background? 

If not, why [don't we block](https://docs.python.org/3/library/subprocess.html#subprocess.Popen.wait) until `hax` process exits?

Reply:
Yes, idea is to run hax in the background.

Reply:
Blocking now after starting hax. To be used as a container entrypoint.